src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -701,22 +702,28 @@ private RequestConfig getRequestConfigWithTimeouts() {
 							log.debug(Utils.PARAM_BOX_FOLDER_ID_STACK + " length="
 									+ boxFolderIdStack.size());
 						} catch (BoxAPIException e) {
-							log.error(this + " boxUploadSiteContent "
-									+ e.getResponse());
-							JSONObject eJSON = new JSONObject(e.getResponse());
-							String errorMessage = eJSON.has("context_info") ? eJSON
-									.getString("context_info") : "";
-									itemStatus
-									.append("Box upload process was stopped due to the following error. Please rename the folder/resource item and migrate site again: \""
-											+ errorMessage + "\"");
-									// the status of file upload to Box
-									MigrationFileItem item = new MigrationFileItem(
-											contentUrl, title, itemStatus.toString());
-									rv.add(item);
-
-									// catch the BoxAPIException e
-									// and halt the whole upload process
-									break;
+							String errorMessage = "There is a problem uploading item " + title + " to Box: "
```

Comment:
I understand your point. However, those messages are for the status report that will be presented to the end user. I do not think the stacktrace is a good fit here...